packages-next/admin-ui/src/pages/ItemPage/index.tsx

```diff
@@ -311,7 +311,9 @@ export const ItemPage = ({ listKey }: ItemPageProps) => {
   let { data, error, loading } = useQuery(query, {
     variables: { id, listKey },
     errorPolicy: 'all',
+    skip: id === undefined,
```

Comment:
This is unrelated but I found it when debugging this. For some reason, Next renders the page without the query object filled so there is an unnecessary request that fails because the id isn't there